doc/ref_model/chapters/chapter05.rst

```diff
@@ -307,23 +424,37 @@ The host profile properties are specified in the following sub-sections. The fol
-infra.hw.cpu.cfg.002 Minimum number of cores per CPU   Specifies the number of cores needed per CPU (*)                                  20    20
-infra.hw.cpu.cfg.003 NUMA alignment                    NUMA alignment enabled and BIOS configured to enable NUMA                         N     Y
-infra.hw.cpu.cfg.004 Simultaneous Multithreading (SMT) SMT enabled that allows each core to work multiple streams of data simultaneously Y     Optional
-==================== ================================= ================================================================================= ===== ================
++----------------------+---------------------------------+---------------------------------+-------------+-------------+
+| Reference            | Feature                         | Description                     | Basic       | High        |
+|                      |                                 |                                 |             | Performance |
++======================+=================================+=================================+=============+=============+
+| infra.hw.cpu.cfg.001 | Minimum number of CPU sockets   | Specifies the minimum number of | 2           | 2           |
+|                      |                                 | populated CPU sockets within    |             |             |
+|                      |                                 | each host (*)                   |             |             |
++----------------------+---------------------------------+---------------------------------+-------------+-------------+
+| infra.hw.cpu.cfg.002 | Minimum number of cores per CPU | Specifies the number of cores   | 20          | 20          |
+|                      |                                 | needed per CPU (*)              |             |             |
++----------------------+---------------------------------+---------------------------------+-------------+-------------+
+| infra.hw.cpu.cfg.003 | NUMA alignment                  | NUMA alignment enabled and BIOS | N           | Y           |
+|                      |                                 | configured to enable NUMA       |             |             |
++----------------------+---------------------------------+---------------------------------+-------------+-------------+
+| infra.hw.cpu.cfg.004 | Simultaneous Multithreading     | Y                               | Optional    |             |
+|                      | (SMT) SMT enabled that allows   |                                 |             |             |
+|                      | each core to work multiple      |                                 |             |             |
+|                      | streams of data simultaneously  |                                 |             |             |
++----------------------+---------------------------------+---------------------------------+-------------+-------------+
```

Comment:
Cells seem to be shifted.

Reply:
Looks okay on my machine :)
![image](https://user-images.githubusercontent.com/11458775/166948039-457868af-1e5b-4ab8-bce4-979f35d010c8.png)

I think it is just some syntax highlight form GitHub.